lib/fog/google/storage.rb

```diff
@@ -4,11 +4,12 @@
 module Fog
   module Storage
     class Google < Fog::Service
-      def self.new(options)
-        if options.keys.join(" ").include? "key"
-          Fog::Storage::GoogleJSON.new(options)
-        else
+      def self.new(options = {})
+        if options.keys.include? :google_storage_access_key_id or
+          (Fog.credentials and Fog.credentials.keys.include? :google_storage_access_key_id)
```

Comment:
I'm still not totally pleased with the result here. If your `~/.fog` file is empty, `Fog.credentials` will throw an error whether you have the options key or not. This is pretty crummy, but I'm not sure how to avoid it without a rescue block or something.